backend/core/Store.qml

```diff
@@ -22,6 +23,14 @@ Item {
         {
             name: qsTr("Compliance Report"),
             page: "../pages/CompliancePage.qml"
+        },
+        {
+            name: "Project / Deploy",
+            page: "../pages/ProjectDeployPage.qml"
+        },
+        {
```

Comment:
can be removed if ProjectParametersPage is merged into ParametersPage as suggested in another file